src/api/departures/schema.ts

```diff
@@ -51,7 +53,9 @@ export const getQuayDeparturesRequest = {
     id: Joi.string().required(),
     numberOfDepartures: Joi.number().default(10),
     startTime: Joi.string(),
-    timeRange: Joi.number().default(86400)
+    timeRange: Joi.number().default(86400),
+    numberOfDeparturesPerLineAndDestinationDisplay: Joi.number()
+
```

Comment:
Do you have Prettier/ESlint enabled in your IDE? I have format on save enabled, and when I work on this file, this line is removed. Also a couple of trailing commas other places in this PR.

Reply:
yep..it was not working as expected. Fixed now.